SECURITY.md

```diff
@@ -4,9 +4,9 @@ Security updates are applied only to the repository itself.
 
 ## Reporting a Vulnerability
 
+Reports are limited to repo matters. Any vulnerability reports related to the addition or removal of PSL entries in the .dat file shall be rejected and referred to filing pull requests that should make mention the alleged urgency.
+
 If you have discovered a security vulnerability in this project, please report it privately. **Do not disclose it as a public issue.** This gives us time to work with you to fix the issue before public exposure, reducing the chance that the exploit will be used before a patch is released.
 Please disclose it at [security advisory](https://github.com/publicsuffix/list/security/advisories/new).
```

Comment:
Please disclose it at [security advisory](https://github.com/publicsuffix/list/security/advisories/new) and send an email with the link to the newly filed issue to <security@mozilla.org> to expedite the review on our end.

Reply:
Done